emperor/support_files/js/emperor.js

```diff
@@ -1815,7 +1818,19 @@ function changeAxesDisplayed() {
   // 		}
 	// }
 
-	var checkedboxes = [];
+    comparisonPositionlength = Object.keys(g_comparisonPositions).length
```

Comment:
I think that makes sense.

If this isn't going to work - is there some test case to show that this is broken?